packages/main/src/AvatarGroup.ts

```diff
@@ -386,7 +391,7 @@ class AvatarGroup extends UI5Element {
 			return this.effectiveDir === "rtl" ? this._getWidthToItem(item) : ltrEffectiveWidth;
 		}
 
-		return button.offsetWidth;
+		return button ? button.offsetWidth : 0;
```

Comment:
Above you have the following early return, 
```js
if (!button) {
	return 0;
}
```
so `return button.offsetWidth` should be fine?